test/helpers/search_helper_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
+
+class RecordHelperTest < ActionView::TestCase
```

Comment:
I think this Class should be `SearchHelperTest`

Reply:
Yay for copypasta. Nice catch, thanks. 🙂